qtpy/tests/utils.py

```diff
@@ -0,0 +1,11 @@
+"""Utility functions for tests."""
+
+import os
+
+
+def using_conda():
+    return os.environ.get('USE_CONDA', 'Yes') == 'Yes'
```

Comment:
```suggestion
    return os.environ.get('USE_CONDA') == 'Yes'
```

Since `os.environ.get` returns `None` if it can't find an env var, I think this is simpler.

Reply:
I left it with a default value (in this case `'Yes'`) to also skip in the cases where the env var is not set but maybe that is not so clear so maybe this should be:

```python
return os.environ.get('USE_CONDA') == 'Yes' or not os.environ.get('USE_CONDA')
```

Reply:
In that case, it would be more idiomatic to reverse them, i.e.

```suggestion
    return not os.environ.get('USE_CONDA') or environ.get('USE_CONDA') == 'Yes'
```

But I'm not sure it is needed, since the behavior of `os.environ,get()` is generally well understood, same as `dict().get()` and `getattr()`.

The other question is whether the tests should skip or not if it can't tell; I'd suggest just going with the status quo, which is to skip, since no one has complained about it.

Reply:
I think changing this, either as @dalthviz or @CAM-Gerlach suggest, would make things harder to understand not simpler, which was the point of my comment.

So I'm fine with leaving this as it is.